bastion/docker-compose.yml

```diff
@@ -1,16 +1,36 @@
-version: '3.9'
+version: "3.9"
+name: vait
+
 services:
+  # Cloudflare tunnel to expose the services
+  cloudflared:
+    image: cloudflare/cloudflared:2024.2.1
+    restart: unless-stopped
+    command: tunnel run
+    environment:
+      - TUNNEL_TOKEN=$CF_TUNNEL_TOKEN
+  # CPU only Ollama instance for light model inference
   ollama:
-    image: ollama/ollama:0.1.27
+    container_name: vait-ollama-cpu
+    image: ollama/ollama:0.1.28
```

Comment:
Do we want to just use the minor version? I feel like pinning down to the patch version is too overkill.

Reply:
> Do we want to just pin to the minor version? I feel like pinning down to the patch version is too overkill.

Welcome to LLM Dockerfile build where major versioning does not exist. I rather not use `latest` or `main` in `production`.

Reply:
See: https://hub.docker.com/r/ollama/ollama/tags